src/scenes/Root/Notifications/Views/Base.tsx

```
@@ -0,0 +1,61 @@
+import { alpha } from '@mui/material/styles';
+import { ElementType } from 'react';
+import { ChildrenProp, extendSx } from '~/common';
+import { RelativeDateTime } from '~/components/Formatters';
+import { NotificationProp } from '../Notification';
+import { ReadIndicator } from '../ReadIndicator';
+
+// Re-export type to simplify imports for concretes which will always need this.
+export type { Views } from '../NotificationViewer';
+
+interface NotificationTypeMap<
+  AdditionalProps = {},
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)